lte/gateway/fabfile.py

```diff
@@ -535,6 +541,13 @@ def _dist_upgrade():
     run('sudo apt-get update')
     run('sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade')
 
+def _from_registry():
+    """
+    Install magma from dev registry
+    """
+    hash = pkg.get_commit_hash('git')
```

Comment:
shouldn't this take the hash as an argument?

Reply:
soon soon with the new public function